src/main/java/org/icepear/echarts/render/Engine.java

```diff
@@ -91,19 +112,17 @@ public void render(String path, Chart<?, ?> chart, String height, String width,
      * @param willOpen whether allowing to open the html in browser automatically
      */
     public void render(String path, Option option, String height, String width, Boolean willOpen) {
-        String jsonStr = new EChartsSerializer().toJson(option);
-        ChartMeta chartMeta = new ChartMeta(height, width, jsonStr);
-        try {
-            Template template = handlebars.compile("index");
-            String html = template.apply(chartMeta);
-            writeHtml(html, path, willOpen);
-        } catch (IOException e) {
-            log.info("render: Handlebars cannot find corresponding templates.");
+        String html = compileHandleBar("index", option, height, width);
+        if (html.equals("")) {
```

Comment:
`compileHandleBar` already handles exceptions, why check the return value here?
I would either expose `IOException` or write empty string without error log.
BTW, the official name is `Handlebars`, you probably want to change the name of `compileHandlerBar`.